### DISCUSSION TITLE
Add support for Flowistry

### DISCUSSION THREAD

**Discussion by kellytk** [helix-editor/helix#9049] [General]
From https://github.com/willcrichton/flowistry:

_Flowistry is a tool that analyzes the [information flow](https://en.wikipedia.org/wiki/Information_flow_(information_theory)) of Rust programs. Flowistry understands whether it's possible for one piece of code to affect another. Flowistry integrates into the IDE to provide a "focus mode" which helps you focus on the code that's related to your current task._

The animated gif helpfully illustrates its functionality.

**Comment by joaogui1** [comment]
Could this be on the editor by default (probably toggable on the rust settings) instead of relegating it to a future plugin? It's a pretty powerful feature
